omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -172,8 +172,8 @@ private void initSwagger() {
 		final Info info = new Info()
 		        .version(OpenmrsConstants.OPENMRS_VERSION_SHORT)
 		        .title("OpenMRS API Docs")
-		        .description("OpenMRS RESTful API specification")
-		        .contact(new Contact().url("http://openmrs.org"))
+		        .description("OpenMRS RESTful API documentation generated by Swagger")
+		        .contact(new Contact().name("OpenMRS Talk").url("https://talk.openmrs.org"))
```

Comment:
What is the reason behind replacing openmrs.org with talk.openmrs.org?

Reply:
As there is where one could get support. If that doesn't make much sense, lets leave it as it is with openmrs.org